### PR TITLE
Add table of which machines C-Star has been tested on

### LINKED ISSUE #186: have a table of which machines we have tried running C-Star on

### PR DESCRIPTION
**Pull request by TomNicholas**
- [x] Closes #186

### CODE REVIEW

File: docs/machines.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Tested Machines
+
+C-Star aims to be deployable on a wide range of HPC systems, but so far it has been tested on a limited number.
+
+| Name of system | Testing Status | Notes |
+| -------------- | -------------- | ----- |
+| Local Mac      | Tested         |       |
```
Comment (dafyddstephenson) [inline docs/machines.md]: Would call this MacOS ARM64 (Apple Silicon)

File: docs/machines.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+| Perlmutter     | Tested         |       |
```
Comment (dafyddstephenson) [inline docs/machines.md]: Would go for NERSC Perlmutter, NCAR Derecho, SDSC Expanse. Would also suggest hyperlinking the names to the machines' homepages:

https://www.sdsc.edu/support/user_guides/expanse.html
https://docs.nersc.gov/systems/perlmutter/architecture/
https://ncar-hpc-docs.readthedocs.io/en/latest/compute-systems/derecho/

**Review by dafyddstephenson — CHANGES_REQUESTED** [review]
Nice one, can't actually see it in situ as it looks like the RTD build is hanging, but these would be my suggestions assuming all looks good in the rendered version

File: docs/machines.md (hunk 2 above)
Comment (TomNicholas) [reply docs/machines.md]: added in https://github.com/CWorthy-ocean/C-Star/pull/190/commits/5cd28f026ce7251a22550369418ca543ea8cb77a

File: docs/machines.md (hunk 1 above)
Comment (TomNicholas) [reply docs/machines.md]: added in https://github.com/CWorthy-ocean/C-Star/pull/190/commits/5cd28f026ce7251a22550369418ca543ea8cb77a

**Pull request merged**